root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -800,6 +805,8 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
                 return id;
             }
         }
+
+        return null;
```

Comment:
`null` *seems* fine, but since our relationship editor code doesn't have Flow types, I'm afraid of possibly breaking conditions that use `===`. So I'd keep this as `undefined` for safety.

```suggestion
        return undefined;
```